backend-api/src/functions/common/session/sessionsDemo.test.ts

```diff
@@ -0,0 +1,70 @@
+import { SessionState } from "./Session";
+import { DynamoDbAdapter } from "../../adapters/dynamoDbAdapter";
+import { randomUUID } from "crypto";
+import "dotenv/config";
+import { BiometricSessionFinished } from "./updateOperations/BiometricSessionFinished";
+import { BiometricTokenIssued } from "./updateOperations/BiometricTokenIssued";
+
+describe("Sessions", () => {
+  // Log into AWS and run this to demonstrate the functionality of the new get/update methods.
```

Comment:
Are there any other pre reqs to running this test? I am currently getting the following error:

ResourceNotFoundException: Requested resource not found

Which is being caught after running:

await this.dynamoDbClient.send(new PutItemCommand(input))